### DISCUSSION TITLE
Title bar no longer in accent color

### DISCUSSION THREAD

**Discussion by ericrechlin** [sumatrapdfreader/sumatrapdf#3881] [General]
In previous (3.4.6 and earlier) versions of Sumatra PDF, the title bar respected the Windows standard, using the accent color (blue in my case) when active and white when inactive, at least when running in Windows 10/11.  In the current version (tested 3.5.2), the title bar is now purely determined by the theme, so it's always white when in light mode, regardless whether it is active.  This makes it very difficult to tell when it's the active window.  Is there any possibility this functionality could be restored, so at least when in light mode it respects the Windows accent color selection when active, like most other programs?  I do not use dark mode so I don't know if its behavior is normal there, but that should be verified also.

**Comment by GitHubRulesOK** [comment]
Its a Widows setting from way back when and still functional.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/30a828a5-3476-4875-a0c9-7042049656b2)

Widows must be using title bar colours and SumatraPDF Advanced set to use those SystemColors=true
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/71139dee-dd60-428d-afab-4f4b5be81932)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/78af1c8b-371d-4804-ac8d-0f70ff3b5664)

**Reply by egossett** [reply]
It would be really nice to have a theme option to use the Windows theme that is currently in effect.

**Comment by ericrechlin** [comment]
It looks like you are using Windows 10.  It's not working for me on Windows 11.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/29085101/9e10de7b-b33c-4642-87f5-486019b1e231)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/29085101/60b4fe39-314e-4ea8-81f7-2478a1c0e3a8)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/29085101/4af0f752-2089-4e0c-9df3-4e291454b1c5)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/29085101/7085cb99-c811-48e3-a623-4eb1ea3aaa26)

Meanwhile, 3.4.6 worked fine for this (the bookmarks tree worked better, too, with full-width selection, but that's a separate issue):
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/29085101/fa800272-ebbd-404e-9359-0716660b4415)

**Comment by henningBunk** [comment]
Love how minimal I can get Sumatra. The only eye sore is the title bar. I am on Windows 11 and the `UseSysColors` has no effect. Dark and Darker mode also did not work. Would be great if you could hide the title bar completly, PotPlayer is able to do this, if you want to move the window around or close it you can hover over the top and than a title bar appears.

But having it dark would already be a big plus.

**Comment by kjk** [comment]
`UseSysColors` is deprecated. It just leads to bug reports that I cannot reproduce because they are caused by different theme setting than I use. It's not worth the time I spent maintaining it.

**Comment by iaselle** [comment]
I have the same problem and the readibility of white tabs on white captions is not good at all.
Using dark theme (and Windows white theme), the active chapter on the lateral bar is black on dark grey, not readable either.
Thanks to this issue, I downgraded to version 3.4.6 and will stick to it ...

**Comment by milnak** [comment]
This is still an issue in 2026. Here's what I see on Windows 11.

Double Commander showing correct color:
<img width="67" height="137" alt="image" src="https://github.com/user-attachments/assets/2adc27f9-1aab-43e3-aba1-8935a5fa56c2" />

which doesnt get applied to sumatrapdf caption when made active.
